apps/backend/src/modules/users/libs/constants/constants.ts

```diff
@@ -1 +1,3 @@
+const ONE_HUNDRED = 100;
```

Comment:
Let's change a bit the name to ONE_HUNDRED_PERCENT_COMPLETION in order to give this constant a purpose because ONE_HUNDERD makes no more sense then 100 a plain number. 
When adding constant try to find the name which will explain the purpose of this constant. You may have a dozen of 100 in your constants file but they may have different names because of different puprose